src/CloudFoundry.Build.Tasks.IntegrationTests/app.config

```diff
@@ -31,8 +31,11 @@
       <setting name="ServerUri" serializeAs="String">
         <value>https://api.1.2.3.4.xip.io</value>
       </setting>
-      <setting name="Route" serializeAs="String">
-        <value>testapp{0}.1.2.3.4.xip.io</value>
+      <setting name="Host" serializeAs="String">
```

Comment:
These new settings must be included in the integration target for build.proj, so they can be automated.